examples/access-token-server/yarn.lock

```diff
@@ -534,7 +534,7 @@
     request "^2.88.0"
     request-promise-native "^1.0.8"
 
-"@looker/sdk@^21.20.1", "@looker/sdk@^22.0.0":
```

Comment:
?

Reply:
Yes I'm looking at that and will figure out before I merge.